client/client-lib/src/lib.rs

```diff
@@ -192,8 +195,9 @@ impl<T: AsRef<ClientConfig> + Clone> Client<T> {
             config: self
                 .config
                 .as_ref()
-                .get_module::<LightningModuleClientConfig>("ln")
-                .expect("needs lightning module client config"),
+                .get_module_by_kind::<LightningModuleClientConfig>("ln")
```

Comment:
Usually on the client we can easily just look up the module by kind, and assume there's only one (or rather: we are going to use the first one found).